lib/PuppeteerSharp/PageAccessibility/AXNode.cs

```diff
@@ -34,7 +33,6 @@ public AXNode(AccessibilityGetFullAXTreeResponse.AXTreeNode payload)
 
             _richlyEditable = payload.Properties.FirstOrDefault(p => p.Name == "editable")?.Value.Value.ToObject<string>() == "richtext";
             _editable |= _richlyEditable;
-            _expanded = payload.Properties.FirstOrDefault(p => p.Name == "expanded")?.Value.Value.ToObject<bool>() == true;
```

Comment:
this one is interesting. should _expanded  actually be used?

Reply:
Nope, It's not exposed by puppeteer.